tests/app/main/test_frameworks.py

```diff
@@ -4821,9 +4525,11 @@ def test_we_abort_if_agreement_does_not_match_supplier_framework(
             supplier_framework
         )
 
+    @mock.patch('app.main.views.frameworks.DMNotifyClient', autospec=True)
```

Comment:
I think we already patch the `send_email` method of the DMNotifyClient in the setup of this test class so we don't need this patch - we can use `self.notify_send_email` in the test.